LayoutTests/imported/w3c/web-platform-tests/html/capability-delegation/delegate-fullscreen-request-subframe-cross-origin.https.sub.tentative-expected.txt

```diff
@@ -1,6 +1,6 @@
-CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: Type error
-CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: Type error
-CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: Type error
+CONSOLE MESSAGE: Unhandled Promise Rejection: TypeError: Cannot request fullscreen without transient activation.
```

Comment:
I think there is a platform-specific expected result for GTK/WPE that needs this same fix.